lib/open_food_network/orders_and_fulfillments_report/customer_totals_report.rb

```diff
@@ -192,7 +192,7 @@ def columns
       def line_item_includes
         [{ variant: [{ option_values: :option_type }, { product: :supplier }],
            order: [:bill_address, :ship_address, :order_cycle, :adjustments, :payments,
-                   :user, :distributor, shipments: { shipping_rates: :shipping_method }] }]
```

Comment:
If we're removing these two includes, we can remove the private method below as well.

Reply:
yeah, I would agree but, I am not sure exactly why, removing it increases the load time on my test in around 10 secs